src/swap.rs

```diff
@@ -25,6 +25,14 @@ pub enum SwapKind {
     Herc20Hbit(SwapParams),
 }
 
+impl From<SwapKind> for SwapParams {
```

Comment:
Because you lose information by doing this conversion I'd been more keen to see
```
impl SwapKind {
	fn params(self) -> SwapParams {
...
	}
}
```

It's a nitpick.

Reply:
Fair enough!

Reply:
Never used anyway. I'm just gonna remove it.